apps/worker/src/app/workflow/services/metric-queue-active.service.ts

```diff
@@ -79,7 +63,7 @@ export class MetricQueueService extends QueueService<Record<string, never>> {
   private getWorkerOpts(): WorkerOptions {
     return {
       ...this.bullConfig,
-      lockDuration: 500,
+      lockDuration: 900,
```

Comment:
i wonder if there were any particular reasons for this change.

Reply:
since the job happens every second I boosted the lock time to 900 seconds so that the job had 900ms to finish before the next one kicks off